supplementary/alica_ros1/supplementary_tests/libalica-supplementary-tests/include/libalica-supplementary-tests/VariableHandling/VHMaster.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <engine/BasicCondition.h>
+#include <engine/BasicConstraint.h>
+#include <engine/BasicPlan.h>
+#include <engine/BasicUtilityFunction.h>
+#include <engine/RunningPlan.h>
+#include <engine/blackboard/Blackboard.h>
+
+#include <boost/dll/alias.hpp>
+
+extern bool vhStartCondition;
```

Comment:
This is weird.  Is this new?  What problem is this trying to solve?

Reply:
Its a boolean value which at some point in one of the tests will be set from false to true to trigger a transition and is a leftover from old code. I can replace it by using the global blackboard instead and remove it.